src/lib.rs

```diff
@@ -697,6 +701,7 @@ impl Uuid {
     pub fn get_version(&self) -> Option<UuidVersion> {
         let v = self.bytes[6] >> 4;
         match v {
+            0 => Some(UuidVersion::Nil),
```

Comment:
If we're going to add an explicit `Nil` variant then I think we should change this match to `0 if self.is_nil()`, since this version is a bit unique in purely describing the value of the uuid. We can prove when the version is not valid. Otherwise:

```rust
let uuid = Uuid::from_uuid_bytes([4, 54, 67, 12, 43, 2, 2, 76, 32, 50, 87, 5, 1, 33, 43, 87]);

assert_eq!(nil.get_version(), Some(UuidVersion::Nil));
```

Is actually true, even though that uuid is definitely not nil.

Reply:
Fixed